data/help/texts.json

```
@@ -244,6 +244,22 @@
   {
     "type": "help",
     "order": 17,
+    "name": "<s|S>: Skills and proficiencies",
+    "messages": [
+      "SKILLS",
+      "Each character has a collection of skills they may learn over time.  These are grouped into four categories: Melee, Ranged, Crafting, and Interaction.  Each skill begins at level 0 and increases with study and practice up to a maximum of level 10.  Each level is harder to attain than the one before, but each level also increases the character's ability to do things using that skill, and may unlock crafting recipes or other actions gated by skill level.  Reaching level 10 in any skill represents true mastery, near the limit of human capability.",
+      "The lower left column of the <press_player_data> screen displays all skills and their current levels, along with percentage progress toward the next level.  Skills highlighted in light red represent those where the character's knowledge has exceeded their practical experience.  This may occur when learning skills primarily from books, without practicing activities or crafts that also exercise those skills.",
+      "Combat skills in the Melee and Ranged categories are quite specialized, focused on specific damage types and firearm flavors.  The Crafting and Interaction categories are more general, each covering a potentially wide range of techniques and abilities.",
+      "PROFICIENCIES",
+      "Separate from Skills, Proficiencies represent a more granular level of character talents.  They are generally mastered over a shorter time span, and their names and purviews tend to be more concrete and specific than those of skills.",
+      "Proficiencies are often defined as a tree of knowledge, where beginners must first master the basic principles before moving onto more advanced topics.  For instance, before a character learns \"Blacksmithing\", they must first acquire \"Principles of Metalworking\".  Once they have learned \"Blacksmithing\", they may dive deeper into \"Armorsmithing\", \"Bladesmithing\", or \"Manual Tooling\".",
+      "The lower right column of the <press_player_data> screen shows each Proficiency, starting in white with those fully learned, followed in grey by those partially learned, and their current progress.  Partially learned proficiencies give some benefit, gradually reducing penalties until they are fully learned.  Once fully learned, proficiencies are considered mastered, and are not forgotten.",
+      "Many crafting recipes involve one or more proficiencies.  When a character has not fully learned all the proficiencies required for crafting, there may be a penalty to crafting time, and/or an increased chance of failure.  These penalties are multiplied when several proficiencies are involved, making some advanced recipes extremely time-consuming and error-prone.  For this reason, it's recommended to learn at least some of the proficiencies with easier recipes before moving onto more advanced multiple-proficiency recipes."
```

```suggestion
      "Many crafting recipes involve one or more proficiencies.  Trying to craft these recipes without the necessary skill set is a long and arduous process with a high rate of failure.  Learning the missing proficiencies offers a bonus to time and to chance of success.  The effect of missing proficiencies are multiplied when several proficiencies are involved, making some advanced recipes extremely time-consuming and error-prone without appropriate knowledge.  For this reason, it's recommended to learn at least some of the proficiencies with easier recipes before moving onto more advanced multiple-proficiency recipes.  Books may also grant a portion of the proficiency bonus."
```
I'd prefer to frame this more as a bonus granted by the proficiency than a penalty.